Core/include/Acts/Seeding/SeedFinderUtils.hpp

```diff
@@ -56,16 +56,16 @@ LinCircle transformCoordinates(external_spacepoint_t& sp,
 /// @param[in] bottom Should be true if vec are bottom spacepoints.
 /// @param[out] linCircleVec The output vector to write to.
 template <typename external_spacepoint_t>
-void transformCoordinates(
+std::vector<std::size_t> transformCoordinates(
```

Comment:
There is now a returned object. Should documentation be updated?

Reply:
updated